addons/mail/static/tests/translation/translation.test.js

```diff
@@ -41,7 +45,11 @@ test("Toggle display of original/translated version of chatter message", async (
     await contains("[title='Revert']");
     await click("[title='Revert']");
     await contains(".o-mail-Message", { text: "Al mal tiempo, buena cara." });
-    await click("[title='Translate']");
+    mockUserAgent("Chrome/0.0.0 Android (OdooMobile; Linux; Android 13; Odoo TestSuite)");
+    expect(isMobileOS()).toBe(true);
+    await hover(".o-mail-Message-actions");
+    await click("[title='Expand']");
+    await click("span", { text: "Translate" });
```

Comment:
We should not hide a mobile test inside a desktop test. 
It should also assert that the text was translated.